src/mpyl/project.py

```diff
@@ -361,7 +361,7 @@ class TraefikHost:
     has_swagger: bool
     tls: Optional[TargetProperty[str]]
     whitelists: TargetProperty[list[str]]
-    priority: Optional[int]
+    priority: TargetProperty[int]
```

Comment:
I think this should be `Optional[TargetProperty[int]]`

Reply:
You're completely right, it should be an optional. I'll apply that change in the next commit 👍